src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1094,6 +1142,9 @@ components:
           example: 01234 567890
         address:
           $ref: '#/components/schemas/Address'
+        delivery:
+          # TODO make delivery as mandatory (EIP1-4668) once changes in VCA (EIP1-4665) are completed. TODO
+          $ref: '#/components/schemas/CertificateDelivery'
```

Comment:
Currently vca-api calls this endpoint to generate AED, therefore initially making it as an optional. Subsequent subtasks are captured in the story to eventually make this field as mandatory once VCA populates this.

Reply:
Currently `vca-api` calls this endpoint to `generate AED`, therefore to not break vca-api, we are initially marking this new attribute as an optional field. 
Subsequent subtasks are captured in the story to eventually make this as `mandatory` field once VCA populates this in [PR](https://github.com/cabinetoffice/eip-ero-voter-card-applications-api/pull/575) .